src/styles/calendar.scss

```diff
@@ -0,0 +1,47 @@
+@use '../styles/variables' as *;
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 726px;
+  gap: 1px;
+  padding: 10px;
+  padding-top: 20px;
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $cell-size;
+    height: $cell-size;
+
+    background-color: $cell-color;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+      }
+    }
+
+    &:hover {
+      translate: 0 -20px;
+      background-color: #ffc0cb;
+      transition: translate, background-color 0.5s, 0.5s;
+      cursor: pointer;
+    }
+
+  }
+
+  @each $day, $day-number in $days {
+    &--start-day--#{$day} > :first-child {
+      margin-left: $first-cell-margin * $day-number + $day-number - 101px;
```

Comment:
what is the `101px`? I think you should use `$cell-size` here and delete `$first-cell-margin` because you want margin to be the same as cell size